packages/elements-angular/src/directives/control-value-accesors/text-value-accessor.directive.ts

```diff
@@ -5,7 +5,7 @@ import { ValueAccessorDirective } from './value-accessor.directive';
 
 @Directive({
   selector:
-    'ino-autocomplete,ino-currency-input,ino-input,ino-markdown-editor,ino-textarea,ino-range,ino-select,ino-datepicker,ino-segment-group',
+    'ino-autocomplete,ino-currency-input,ino-input:not([type=number]):not([type=file]),ino-markdown-editor,ino-textarea,ino-select,ino-datepicker,ino-segment-group',
```

Comment:
For using multiply `:not()` selectors I suggest writing it like here:
https://developer.mozilla.org/en-US/blog/css-not-pseudo-multiple-selectors/#negating_multiple_selectors

Reply:
I've just tried to add some simple jest test to the package but I think as long as we've got these conflicts with stencils outdated Jest version, I don't see us adding any new tests soon. I'll write that down in a ticket.